File: examples/cache/decrement/go/decrementAValue.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/cache"
)

// Decrement a value (if it's a number)
func DecrementAvalue() {
	cacheService := cache.NewCacheService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cacheService.Decrement(&cache.DecrementRequest{
		Key:   "counter",
		Value: 2,
	})
	fmt.Println(rsp, err)
}

File: examples/js/cache/decrement/decrementAValue.js
const { CacheService } = require("m3o/cache");

// Decrement a value (if it's a number)
async function decrementAvalue() {
  let cacheService = new CacheService(process.env.MICRO_API_TOKEN);
  let rsp = await cacheService.decrement({
    key: "counter",
    value: 2,
  });
  console.log(rsp);
}

decrementAvalue();
